docs/pubsub-usage.rst

```diff
@@ -66,6 +66,23 @@ Delete a topic:
    >>> topic = client.topic('topic_name')
    >>> topic.delete()  # API request
 
+Fetch the IAM policy for a topic:
```

Comment:
I thought @jonparrott said IAM was at the service level rather than the object level?